advocacy_docs/partner_docs/PreciselyConnectCDC/04-Configuratingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Configuration'
+description: 'Walkthrough on configuring the integration'
+---
+
+## Prerequisites
+
+Implementing Precisely Connect CDC with EDB Postgres Advanced Server requires the following components:
+
```

Comment:
use distributions:

Implementing EDB software with Precisely Connect CDC requires the following components:

Note
The EDB Postgres Advanced Server and PostgreSQL Server products will be referred to as Postgres Distribution. The specific Distribution type will be dependent upon customer need or preference.

Postgres Distribution
Precisely Connect CDC software

Reply:
As per the Doc @scottatchue provided changed following 
1. Product Description
2. Configuration Page
3. Using Page.